db/schema.rb

```diff
@@ -98,6 +99,11 @@
     t.string   "last_sign_in_ip"
     t.string   "phone"
     t.string   "skypename"
+    t.text     "preference"
```

Comment:
Although you have removed the preference migration, the schema has not been updated yet. Please run `rake db:migrate` and then commit the schema file again.